core/domain/summary_services_test.py

```diff
@@ -196,6 +196,97 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
         self.assertDictContainsSubset(expected_summary,
                                       displayable_summaries[0])
 
+    def test_get_displayable_exp_summary_dicts(self):
+        exp_summary_input = exp_services.get_exploration_summaries_matching_ids(
+            [self.EXP_ID_2])
+        exp_summary_input.insert(0, None)
```

Comment:
Ah, sorry. How about

 [None] + exp_services.get_exploration_summaries_matching_ids(
[self.EXP_ID_2])?

On Tuesday, April 19, 2016, S Giritheja <notifications@github.com> wrote:

> In core/domain/summary_services_test.py
> <https://github.com/oppia/oppia/pull/1746#discussion_r60357523>:
>
> > @@ -196,6 +196,97 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
> >          self.assertDictContainsSubset(expected_summary,
> >                                        displayable_summaries[0])
> >
> > +    def test_get_displayable_exp_summary_dicts(self):
> > +        exp_summary_input = exp_services.get_exploration_summaries_matching_ids(
> > +            [self.EXP_ID_2])
> > +        exp_summary_input.insert(0, None)
>
> Here exp_services.get_exploration_summaries_matching_ids(
> [self.EXP_ID_2])
> returns a list and we need to insert a None object to that list. The
> method you suggested will create a list in a list.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/oppia/oppia/pull/1746/files/c753745b24c841e0e2c878b41ecaabde3b3168eb#r60357523>
>